ParrelSync/Editor/ClonesManager.cs

```diff
@@ -551,6 +551,12 @@ public static List<string> GetCloneProjectsPath()
             /// Copy all files from the source.
             foreach (FileInfo file in source.GetFiles())
             {
+                // Ensure file exists before continuing.
+                if (file.Exists)
```

Comment:
I was running into a `FileNotFoundException` when trying to "Add new clone" (I'm on 2020.3.22f1) and the process would hang indefinitely.

To get it to work I applied the changes in this PR and changed this to
`if (!file.Exists) {
  continue;
}`

Because we _don't_ want to move through the rest of the loop if the file doesn't exist.  `continue` will move to the next cycle of the loop.

I didn't do research on why this file didn't exist - but the solution seems to have gotten me past my problem and I'm able to Add a new clone.

Reply:
@Ajackster wow that's what I get for writing changes in the GitHub editor. Thanks & nice spot.